tilewe/__init__.py

```diff
@@ -177,15 +209,45 @@ def __init__(self, name: str, rot: _PieceRotation, pt: Tile):
         self.adjacent = sorted(list(self.adjacent))
         self.corners = sorted(list(self.corners))
 
+# internal global data for all game pieces
+# initialized on library load one time below
+PIECE_COUNT = 0
+_PIECES: list[_Piece] = []
+_PIECE_ROTATIONS: list[_PieceRotation] = []
+_PIECE_ROTATION_POINTS: list[_PieceRotationPoint] = []
+_PRP_SET_ALL: _PrpSet = 0
```

Comment:
moved below the internal class declarations so they can be accessed as types without string trickery